android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -1995,6 +1995,7 @@ public void setFullscreen(boolean fullscreen) {
             if (controls && fullScreenPlayerView != null) {
                 fullScreenPlayerView.dismiss();
                 reLayout(exoPlayerView);
+                reLayout(playerControlView);
```

Comment:
Thank you for the fix, I think this is a correct change to apply.
BTW can I propose you additional modifications:
- I think there can be some corner case where  playerControlView can be null, can you please test it before accessing to it
-  There is another call to 'reLayout(exoPlayerView);' in initializePlayerSource I think it make sens to apply this change at this place also.
- Then these 2 relayout calls should be put in a function :)
Thank you !

Reply:
> I think there can be some corner case where playerControlView can be null, can you please test it before accessing to it

in `reLayout` we are checking if passed view is not null

> - There is another call to 'reLayout(exoPlayerView);' in initializePlayerSource I think it make sens to apply this change at this place also.
> - Then these 2 relayout calls should be put in a function :)
Thank you !
Done ✅